src/code/game.c

```diff
@@ -334,14 +351,25 @@ void GameState_InitArena(GameState* gameState, size_t size) {
     void* arena;
 
     PRINTF("ハイラル確保 サイズ＝%u バイト\n"); // "Hyrule reserved size = %u bytes"
+
+#ifdef OOT_DEBUG
     arena = GameAlloc_MallocDebug(&gameState->alloc, size, "../game.c", 992);
+#else
+    arena = GameAlloc_Malloc(&gameState->alloc, size);
+#endif
```

Comment:
Could you add macros for this and LogUtils_HungupThread at https://github.com/zeldaret/oot/blob/0e6dc594e93fdd6ee17bf16cd63918986dc2589b/include/macros.h#L193? The GameAlloc_Malloc one is only used here but it fits the others, while the LogUtils_HungupThread will be needed at least in z_play later